client/gulpfile.js

```diff
@@ -55,6 +66,31 @@ gulp.task('stylelint', () => {
     }))
 })
 
+gulp.task('svgstore', function () {
+  let svgs = gulp.src(sources.images.svg.all)
```

Comment:
True. I'm fairly sure all `let`s in the gulpfile can be `const`s at present. Will update.